docs/book.toml

```diff
@@ -6,4 +6,9 @@ src = "src"
 title = "The Fe Guide"
 
 [output.html]
-additional-css = ["theme/reference.css"]
\ No newline at end of file
+additional-css = ["theme/reference.css"]
+
+[output.linkcheck]
+follow-web-links = true
+warning-policy = "error"
+exclude = [ 'github\.com' ]
```

Comment:
I excluded `github.com` here because there are so many links to `github.com` in the book, so `github.com` refuses requests, and the test would fail.